gensim/models/ldamodel.py

```diff
@@ -800,15 +800,24 @@ def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
             chosen_topics = sorted_topics[:num_topics // 2] + sorted_topics[-num_topics // 2:]
 
         shown = []
+
+        topic = self.state.get_lambda()
         for i in chosen_topics:
             if formatted:
```

Comment:
Too much code duplication
Just add an extra `if formatted` block in the end that changes formatting